website/content/docs/upgrading/index.mdx

```diff
@@ -41,9 +46,40 @@ considerations:
   is designed for you to know when and how to safely upgrade the server.
 
 Please see our [compatibility promise](/docs/upgrading/compatibility)
-for details on how to determine compatbility. This page will focus on
+for details on how to determine compatibility. This page will focus on
 how to upgrade components in both compatible and incompatible scenarios.
 
+### Server Upgrade Command
+
+As of 0.2.1, Waypoint provides a command for users to automatically upgrade
```

Comment:
"automatically" implies hands-off to me; but the user still has to take action here and run the command, as they can't set the server to upgrade automatically on new releases.
```suggestion
As of 0.2.1, Waypoint provides a command for users to upgrade the
```

Reply:
Hm, makes sense to me. I guess it's less automatic and more _automated_. So leaving it off makes sense :+1: